### PR TITLE
Change container to images

### PR DESCRIPTION
**Pull request by peninoule**
I saw an error in the index.md file, line 52.

The inital file was saying "... we saw that each container is effectively read-only"

I would like to change this to "... each images is effectively read-only"

Best regards
peninoule

### CODE REVIEW

**Comment by mikesir87** [comment]
Hi @peninoule and thanks for the PR and feedback! 👋  

I see what you're saying, as it is a little confusing. The container itself isn't read-only, but images are. But we're trying to show that containers start from scratch each time they restart, rather than having things persist. So, maybe the best approach might be to reword that entire section, rather than a single word change. That sound ok?

File: docs_en/tutorial/persisting-our-data/index.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ What you'll see is that the files created in one container aren't available in a
 
 ## Container Volumes
 
-With the previous experiment, we saw that each container is effectively read-only. While containers
+With the previous experiment, we saw that each images is effectively read-only. While containers
```
Comment (mikesir87) [inline docs_en/tutorial/persisting-our-data/index.md]: How about something like this? Thoughts?

```suggestion
With the previous experiment, we saw that each container restarts from scratch from its image. While containers
```

**Comment by peninoule** [comment]
Hello Mickael,

Thanks for the answer ! :)

I see the point now !

I’m no docker expert ( just a student from Rachid Zarouali’s classes), so I’m not sure if I can really give advices to someone like you but still, your correction seems to clarify the situation and I quite like it !

Best regards,
Pierre-Jean


De : Michael Irwin<mailto:notifications@github.com>
Envoyé le :jeudi 28 novembre 2019 18:25
À : dockersamples/101-tutorial<mailto:101-tutorial@noreply.github.com>
Cc : CROUZET Pierre-Jean<mailto:pierrejean.crouzet@ipilyon.net>; Mention<mailto:mention@noreply.github.com>
Objet :Re: [dockersamples/101-tutorial] Change container to images (#11)


@mikesir87 commented on this pull request.


In docs_en/tutorial/persisting-our-data/index.md<https://github.com/dockersamples/101-tutorial/pull/11#discussion_r351887671>:

> @@ -49,7 +49,7 @@ What you'll see is that the files created in one container aren't available in a







 ## Container Volumes







-With the previous experiment, we saw that each container is effectively read-only. While containers



+With the previous experiment, we saw that each images is effectively read-only. While containers



How about something like this? Thoughts?
⬇️ Suggested change

-With the previous experiment, we saw that each images is effectively read-only. While containers



+With the previous experiment, we saw that each container restarts from scratch from its image. While containers

**Comment by mikesir87** [comment]
Sorry for the delay! But no worries about not being an expert. I've been doing it for so long that I sometimes forget how much there is to learn. So, your perspective is definitely valuable here! If you think that change will work, can you accept/commit the suggestion? Then, I'll merge the PR. 👍

**Comment by mikesir87** [comment]
Thanks! I've merged it and will get it built/pushed out shortly 👍

**Pull request merged**